### PR TITLE
Fix 1088 Message displayed when user is successfully signed in

### LINKED ISSUE #1088: message should be displayed when user is successfully logged in
**Summary:**

There should be a message when passcode is successfully saved and user is logged in.

**Steps to reproduce:**

Log in using a passcode

**Expected behaviour:**

When passcode is successfully entered there should be a message. 

**Observed behaviour:**

There is no message displayed when a passcode is successfully entered

**Device and Android version:**

Android Version:- 8.1.0

### PR DESCRIPTION
**Pull request by theabhishekavi**
Fixes #1088 

Screenshots
Alt Text

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by theabhishekavi** [comment]
@ShivangiSingh17 can you help why there is travis ci check error?

**Comment by aggarwalpulkit596** [comment]
screenshot 2019-03-04 at 11 41 28 am
@theabhishekavi here you go

**Comment by theabhishekavi** [comment]
yes,i got that.thanks! @aggarwalpulkit596

**Comment by miPlodder** [comment]
@theabhishekavi Create a PR on this [Library REPO](https://github.com/openMF/mifos-passcode) and make changes there accordingly.

File: app/src/main/java/org/mifos/mobilebanking/ui/activities/PassCodeActivity.java [hunk 1]
```diff
@@ -46,6 +47,7 @@ public int getLogo() {
 
     @Override
     public void startNextActivity() {
+        Toast.makeText(this, "Logged in succesfully", Toast.LENGTH_SHORT).show();
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobilebanking/ui/activities/PassCodeActivity.java:50]: Don't use hard-coded strings. Write them in strings.xml file and use them. (Do this in the repo that I have commented on the PR)

**Comment by therajanmaurya** [comment]
Shifted to Library Repo: https://github.com/openMF/mifos-passcode/issues/24

**Pull request closed without merging**